lib/view_component/storybook/controls/control_config.rb

```diff
@@ -30,8 +30,8 @@ def value_from_param(param)
           param
         end
 
-        def component_params
-          @component_params ||= component && component.instance_method(:initialize).parameters.map(&:last)
+        def component_param_names
+          @component_param_names ||= component_params&.map(&:last)
```

Comment:
I know I wrote it this way but does this method need to be public?

Reply:
we do! because of the call in `validates :param, inclusion: { in: ->(control_config) { control_config.component_param_names } }, if: :should_validate_params?`